### ISSUE TITLE
table of parameters

### ISSUE THREAD

**Issue by matt-long** [marbl-ecosys/cesm2-marbl#49]
I would like to include a table of model parameters in the paper. 

In the Latex document we have a table:
```latex
\begin{table}[p!]
\caption{Model parameters (from pandas).}
\centering
\begin{tabular}{lrll}
\toprule
                          Parameter &  Value &              Units &                         Description \\
\midrule
   $\mu_{\mathrm{ref},\mathrm{sp}}$ &    5.0 &  $\mathrm{d^{-1}}$ &  Maximum C-spec growth rate at Tref \\
 $\mu_{\mathrm{ref},\mathrm{diat}}$ &    5.0 &  $\mathrm{d^{-1}}$ &  Maximum C-spec growth rate at Tref \\
 $\mu_{\mathrm{ref},\mathrm{diaz}}$ &    2.5 &  $\mathrm{d^{-1}}$ &  Maximum C-spec growth rate at Tref \\
\bottomrule
\end{tabular}
\end{table}
```

The "from pandas" leads me to believe this was auto generated. Can we revive this workflow and expand to include other parameters?

**Comment by mnlevy1981** [comment]
There's an open issue in the main MARBL repo: https://github.com/marbl-ecosys/MARBL/issues/341 -- I thought https://github.com/marbl-ecosys/MARBL/pull/346 was the PR to bring in the tool, but a quick glance at the files that changed don't include the pandas script so I'll need to keep digging. The first comment in that PR mentions that those changes are necessary for generating the latex table, so I think I'm close to finding the script. (It looks like the MARBL PR will need a companion POP PR, which is why it hasn't been merged yet).

**Comment by mnlevy1981** [comment]
#37 (in this repo) has the [notebook](https://github.com/mnlevy1981/cesm2-marbl/blob/marbl_settings_table/notebooks/MARBL%20settings%20table.ipynb) that relies on the MARBL branch to generate the table as well as a [YAML file](https://github.com/mnlevy1981/cesm2-marbl/blob/marbl_settings_table/notebooks/MARBL_settings.yaml) to control what variables are in the table.

**Comment by matt-long** [comment]
Great! Let me know when that PR is ready—or just merge it. We could merge the basic capability first, then add to the table later. As a first cut, I would like to include all the parameters mentioned in Section 2 of the text.

**Comment by mnlevy1981** [comment]
should've closed this in May when #37 was merged...
